schema.cql

```diff
@@ -0,0 +1,10 @@
+CREATE KEYSPACE IF NOT EXISTS 'photo_gallery';
```

Comment:
Does `'photo_gallery'` need to be in quotes?

Reply:
Made the update. Thanks!